features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/SeekBarWireframeMapperTest.kt

```diff
@@ -60,7 +59,9 @@ internal class SeekBarWireframeMapperTest : AbstractWireframeMapperTest<SeekBar,
     @IntForgery(min = 512, max = 65536)
     var fakeMaxValue: Int = 0
 
-    @FloatForgery(min = 0f, max = 1f)
+    @IntForgery(min = 0, max = 100)
+    var fakeProgressInt: Int = 0
```

Comment:
minor: `fakeProgressInt` is not used anywhere, so maybe you can just remove property and `Forgery` annotation and inject `forge` in `set up` method.

Reply:
Yes I can, I just have a question if I do so, does the `forge` instance of `set up` method uses the same seed as the property forgery? if not, it's not convenient to reproduce the failure.

Reply:
yes, it is the same seed

Reply:
it is just `fun setUp(forge: Forge)` and you'll get the necessary instance.

Reply:
updated